src/D_configure_taskserver.sh

```diff
@@ -1,197 +1,198 @@
 #!/bin/sh
-mkdir_taskddata() {
-	TASKDDATA=$1
-	output=$(mkdir -p $TASKDDATA)
-	echo $output
+mkdir_taskd_data() {
+	TASKD_DATA=$1
```

Comment:
I personally think `TASKD_DATA` is more clear than `TASKDDATA`, so I would also appreciate your change. However, it is more important that this project is compliant with the Taskwarrior organisation. There, they use `TASKDDATA` (as seen here: https://taskwarrior.org/docs/taskserver/configure.html), so if other people use this code, they should not get some error somewhere because of an inconsistent variable name.

So if you would perhaps be so kind to revert `TASKD_DATA`, to `TASKDDATA`, that would be greatly appreciated!

(On a side note, in the future I should remove the `TASKDDATA=$1` alltogether, because `TASKDDATA` is supposed to be a globel variable that is exported with:
```
export TASKDDATA=/var/taskd
```
somewhere early on in the installation. Additionally any parameter that is incoming into a function, e.g.
```
somevariable=$1337
```
Should not be capitalised I think, because:
1. I think they are local variables
2. And https://google.github.io/styleguide/shellguide.html#s7.4-source-filenames says Constants and Environment Variable Names should be capitalized. I think that implies other things should not be capitalised (though this is not stated explicitly).
3. The example of local variables is not capitalized: https://google.github.io/styleguide/shellguide.html#s7.6-use-local-variables